.github/workflows/cs.yaml

```diff
@@ -28,10 +28,10 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@ee0669bd1cc54295c223e0bb666b733df41de1c5 # v2.7.0
```

Comment:
what is the reason?

Reply:
Security. I plan to do one on the main repo too.

Essentially tags/branches are moving targets which makes it a pretty desirable target to exploit.

See https://github.com/PHPCSStandards/PHPCSDevTools/pull/197 which was the initial discussion that made me aware of it.

Also note that Dependabot manages the update (comment included) just fine.